src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -84,6 +84,13 @@
         type="literal",
         trim_chars=('"',),
     ),
+    SingleQuotedLiteralSegment=NamedParser(
+        "single_quote",
+        CodeSegment,
+        name="quoted_literal",
+        type="literal",
+        trim_chars=("'",),
```

Comment:
I'm not familiar with `trim_chars`. What does it do, and why is it needed here?

Reply:
Direct copy from other simile functions. It basicaly takes the starting quote and ending quote marks off IIRC leaving the actual value.